traveler-swift-core/TravelerKit/Models/SearchQuery.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  SearchQuery.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-10-05.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+/// An enum that encompasses all possible search queries
+public enum SearchQuery {
+    case booking(query: BookingQuery)
+    case parking(query: ParkingQuery)
```

Comment:
Yeah using this enum I dont think we need two diff attributes in the model above..

remove the labels too...its clear what they are